### PR TITLE
Export championships

### PR DESCRIPTION
**Pull request by Baiqiang**
Export championships so that the community can use this information directly instead of ugly hard code.

### CODE REVIEW

**Comment by jfly** [comment]
Thanks for taking this on! This actually fixes #1796 (which I've assigned to you).

I think we should include the `eligible_country_iso2s_for_championship` table in here, otherwise people won't know how to handle `greater_china`.

Also, could you update the [README](https://github.com/thewca/worldcubeassociation.org/blob/master/webroot/results/admin/export/template.README.txt) with an explanation about how to use the `championships` table and how it interacts with the `eligible_country_iso2s_for_championship` table?

File: webroot/results/admin/export_public.php [hunk 1]
```diff
@@ -44,6 +44,10 @@
                               WHERE Competitions.showAtAll=1
                               GROUP BY Competitions.id',
     'Scrambles'   => '*',
+    'championships' => 'SELECT championships.competition_id, championships.championship_type
```
Comment (jonatanklosko) [inline webroot/results/admin/export_public.php]: Why would we exclude the `id` field here? I guess it seems useless, but you never know ^^ @jfly?

Comment (jfly) [reply webroot/results/admin/export_public.php]: I have a slight preference for keeping things as similar to the current database schema as possible, so I'd vote for including the id here.

Comment (jonatanklosko) [reply webroot/results/admin/export_public.php]: Nit: simplify to `SELECT championships.* FROM`, we don't want the competitions stuff =)

**Comment by Baiqiang** [comment]
I've updated it. Is some line of the explanation too long? @jfly

File: webroot/results/admin/export/template.README.txt [hunk 2]
```diff
@@ -56,6 +58,11 @@ that no longer exist.  The ISO2 column should reflect ISO 3166-1 alpha-2
 country codes, for countries that have them.  Custom codes may used in some
 circumstances.
 
+eligible_country_iso2s_for_championship stored the special championship types which were neither a country nor a continent.
```
Comment (jfly) [inline webroot/results/admin/export/template.README.txt]: nit: I think I'd rephrase this a bit. How does the following sound?

> eligible_country_iso2s_for_championship stores information about which citizenships are eligible to win special cross-country championship types.
